app/javascript/components/layout/Header.js

```diff
@@ -21,73 +21,69 @@ class Header extends React.Component {
   render() {
     return (
       <React.Fragment>
-        <Navbar bg="primary" variant="dark" expand="lg" className={this.props.show_demo_warning ? '' : 'mb-3'}>
+        <Navbar bg="primary" variant="dark" expand="lg" className={this.props.show_demo_warning ? 'pt-0' : 'mb-3 pt-0'}>
           <Navbar.Brand href={this.props.report_mode ? '/' : this.props.root}>
             Sara Alert<small className="nav-version ml-1">{this.props.version}</small>
           </Navbar.Brand>
           {this.props.current_user && (
             <React.Fragment>
-              <Nav className="pt-2 mr-auto" activeKey={this.getActiveTabKey()}>
+              <Nav className="mt-2 mr-auto" activeKey={this.getActiveTabKey()}>
                 {this.props.current_user?.can_see_enroller_dashboard_tab && (
                   <Nav.Link className="ml-3" href={`${window.BASE_PATH}/patients`}>
-                    <i className="fas fa-table fa-fw mr-2"></i>Enroller Dashboard
+                    <i className="fas fa-table fa-fw nav-icon mr-2"></i>Enroller Dashboard
                   </Nav.Link>
                 )}
                 {this.props.current_user?.can_see_monitoring_dashboards_tab && (
                   <Nav.Link className="ml-3" href={`${window.BASE_PATH}/public_health`}>
-                    <i className="fas fa-table fa-fw mr-2"></i>Monitoring Dashboards
+                    <i className="fas fa-table fa-fw nav-icon mr-2"></i>Monitoring Dashboards
                   </Nav.Link>
                 )}
                 {this.props.current_user?.can_see_admin_panel_tab && (
                   <Nav.Link className="ml-3" href={`${window.BASE_PATH}/admin`}>
-                    <i className="fas fa-user-cog fa-fw mr-2"></i>Admin Panel
+                    <i className="fas fa-user-cog fa-fw nav-icon mr-2"></i>Admin Panel
                   </Nav.Link>
                 )}
                 {this.props.current_user?.can_see_analytics_tab && (
                   <Nav.Link className="ml-3" href={`${window.BASE_PATH}/analytics`}>
-                    <i className="fas fa-chart-pie fa-fw mr-2"></i>Analytics
+                    <i className="fas fa-chart-pie fa-fw nav-icon mr-2"></i>Analytics
                   </Nav.Link>
                 )}
               </Nav>
-              <Form inline className="ml-auto">
-                {!this.props.report_mode && (
```

Comment:
I didn't explicitly add this check back in when cleaning this up because it should already be handled by the `current_user` check on line 28. @holmesie Let me know if I'm missing something and this is actually needed for a reason I may not be thinking of.